docs/api/session.md

```diff
@@ -180,6 +180,67 @@ Emitted when a hunspell dictionary file download fails.  For details
 on the failure you should collect a netlog and inspect the download
 request.
 
+#### Event: 'select-hid-device'
+
+Returns:
+
+* `event` Event
+* `deviceList` [HIDDevice[]](structures/hid-device.md)
+* `webContents` [WebContents](web-contents.md)
```

Comment:
This should be in a `details` struct per our [design guidelines](https://github.com/electron/governance/blob/master/wg-api/best-practices.md#how-will-this-api-be-extended-in-the-future)